sunflower-land/sunflower-land
# [FIX] Squirrel replenish boost

# Description

- Fix Squirrel monkey boost on orange fruit after harvest (replenish time).
- Add Squirrel monkey boost on harvest test suites
- Fix typo in plant fruit test suites

Fixes Discord bug [Link](https://discord.com/channels/880987707214544966/1071705904501768202)

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

Manual testing see video:
- Planting Orange fruit _with_ Squirrel Monkey placed in the farm
- Planting Blueberry fruit _with_ Squirrel Monkey placed in the farm
- Planting Orange fruit _without_ Squirrel Monkey placed in the farm

https://user-images.githubusercontent.com/12151051/216883544-b0e142da-a9b8-4712-a686-94f316118c97.mp4


With adjusted parameter to save time
- Change Blueberry Fruits harvest & replenish time into `6 seconds`
- Change Orange Fruits harvest & replenish time into `6 seconds`

Running the harvest test suites with newly added test suites:
![image](https://user-images.githubusercontent.com/12151051/216883828-1153aac8-fa12-4000-96b7-1c589fc0c395.png)


# Checklist:
